src/membership.md

```diff
@@ -36,18 +36,22 @@ Lang team members can propose new additions to the team as follows:
 These are the questions we ask ourselves when deciding whether someone would be a good choice as a lang team member.
 
 * Has this person demonstrated **strong language design skills**?
-    * Have they led an impactful initiative to completion?
-    * Have they identified a criticial flaw in someone else's design that led to the design being altered?
+    * Have they made major contributions to an area of the language, such as leading an impactful initiative to completion?
+    * Are they able to identify flaws in a design and, just as importantly, come up with creative solutions?
 * Is this person **responsible**?
     * When they agree to take on a task, do they either get it done or identify that they are not able to follow through and ask for help?
 * Is this person able to **lead others to a productive conversation**?
     * Are there times when a conversation was stalled out and this person was able to step in and get the design discussion back on track?
         * This could have been by suggesting a compromise, but it may also be by asking the right questions or encouraging the right tone.
-* Is this person able to **disagree productively**?
+* Is this person able to **disagree constructively and empathically**?
```

Comment:
```suggestion
* Is this person able to **disagree constructively, with empathy**?
```

Reply:
I like the word collaboratively! Nice.

Reply:
I wonder if we should also include the language "treasure dissent"

Reply:
@nikomatsakis I would love to see "treasure dissent" included if you can find a good place for it.